spec/models/reading_spec.rb

```diff
@@ -41,5 +41,21 @@
       })
       expect(@reading.temp).to eq 57
     end
+
+    it "handles nil temperature properly", :vcr do
+      allow(WeatherMan).to receive(:outdoor_temp_for).and_return(nil)
+      user = create(:user, zip_code: '10216')
+      user.sensors.create(name: "abcdefgh")
+      params = ActionController::Parameters.new({
+        "reading" => {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.